packages/user-ui/src/components/input/mod.rs

```diff
@@ -2,6 +2,7 @@ use dioxus::prelude::*;
 
 #[component]
 pub fn InputBox(
+    #[props(default = 0)] id: i64,
```

Comment:
why does id is i64?
Usually, describing only id or index in `id` attribute is anti-pattern.